--- streamripper-1.64.6/lib/http.c.orig	2009-04-27 23:34:12.000000000 +0200
+++ streamripper-1.64.6/lib/http.c	2009-04-27 23:34:19.000000000 +0200
@@ -228,7 +228,7 @@
 
     /* This is the header suggested Florian Stoehr */
     snprintf(buffer, MAX_HEADER_LEN + MAX_HOST_LEN + SR_MAX_PATH,
-	     "GET %s HTTP/1.1\r\n"
+	     "GET %s HTTP/1.0\r\n"
 	     "Accept: */*\r\n"
 	     "Cache-Control: no-cache\r\n"
 	     "User-Agent: %s\r\n"
